src/nlp/load.py

```diff
@@ -506,7 +507,7 @@ def load_dataset(
         **config_kwargs (Optional ``dict``): keyword arguments to be passed to the ``nlp.BuilderConfig`` and used in the ``nlp.DatasetBuilder``.
 
     Returns:
-        ``nlp.Dataset`` or ``Dict[nlp.Split, nlp.Dataset]``
+        ``nlp.Dataset`` or ``nlp.DatasetDict``
             if `split` is not None: the dataset requested,
             if `split` is None, a `dict<key: nlp.Split, value: nlp.Dataset>` with each split.
```

Comment:
Update this line as well `a `dict<key: nlp.Split, value: nlp.Dataset>` with each split` to mention nlp.DatasetDict maybe